frontend/www/js/omegaup/components/contest/Mine.vue

```diff
@@ -0,0 +1,271 @@
+                <a v-bind:href="`/contest/${contest.alias}/edit/`">
+                  <font-awesome-icon
+                    v-bind:title="T.wordsEdit"
+                    v-bind:icon="['fas', 'edit']"
+                  />
+                </a>
+                <a class="ml-2" v-bind:href="`/arena/${contest.alias}/admin/`">
+                  <font-awesome-icon
+                    v-bind:title="T.contestListSubmissions"
+                    v-bind:icon="['fas', 'tachometer-alt']"
+                  />
+                </a>
+                <a
+                  class="ml-2"
+                  v-bind:href="`/contest/${contest.alias}/stats/`"
+                >
+                  <font-awesome-icon
+                    v-bind:title="T.profileStatistics"
+                    v-bind:icon="['fas', 'chart-bar']"
+                  />
+                </a>
+                <a
+                  class="ml-2"
+                  v-bind:href="`/contest/${contest.alias}/activity/`"
+                >
+                  <font-awesome-icon
+                    v-bind:title="T.wordsActivityReport"
+                    v-bind:icon="['fas', 'clock']"
+                  />
+                </a>
+                <a class="ml-2" v-bind:href="`/arena/${contest.alias}/print/`">
+                  <font-awesome-icon
+                    v-bind:title="T.wordsPrintableVersion"
+                    v-bind:icon="['fas', 'print']"
+                  />
+                </a>
+                <a
+                  class="ml-2"
+                  href="#"
+                  v-on:click="onDownloadCsv(contest.alias)"
+                >
+                  <font-awesome-icon
+                    v-bind:title="T.contestDownloadListOfUsersInContest"
+                    v-bind:icon="['fas', 'file-download']"
+                  />
+                </a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Emit } from 'vue-property-decorator';
+import { omegaup } from '../../omegaup';
+import { types } from '../../api_types';
+import T from '../../lang';
+import * as UI from '../../ui';
+import * as time from '../../time';
+
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+import {
+  faEdit,
+  faLink,
+  faTachometerAlt,
+  faChartBar,
+  faClock,
+  faPrint,
+  faFileDownload,
+} from '@fortawesome/free-solid-svg-icons';
+library.add(
+  faEdit,
+  faLink,
+  faTachometerAlt,
+  faChartBar,
+  faClock,
+  faPrint,
+  faFileDownload,
+);
+
+@Component({
+  components: {
+    FontAwesomeIcon,
+  },
+})
+export default class List extends Vue {
+  @Prop() contests!: types.ContestListItem[];
+  @Prop() privateContestsAlert!: boolean;
+
+  T = T;
+  UI = UI;
+  time = time;
+  shouldShowAllContests = false;
+  allContestsVisibilityOption = 'none';
+  selectedContests = [];
```

Comment:
se le puede agregar tipo a esto?

```suggestion
  selectedContests = <string[]>[];
```